lib/ears.rb

```diff
@@ -8,11 +8,9 @@
 module Ears
   class << self
     # The global configuration for Ears.
-    #
+    # @attribute [r] configuration
     # @return [Ears::Configuration]
-    def configuration
-      @configuration ||= Ears::Configuration.new
-    end
+    attr_reader :configuration
```

Comment:
let's stick with the previous interface for configuration https://github.com/ivx/ears/pull/37